src/incident_reports/configs/init_mongo_prod.js

```diff
@@ -0,0 +1,10 @@
+db = db.getSiblingDB('test_concierge');
+
+db.createUser({
```

Comment:
Likewise, we don't need to worry about additional database permissions when using a single-db implementation